tests/integration/pkg/cluster/clusters.go

```diff
@@ -286,6 +316,37 @@ func getPodLogs(clients *clients.Clients, podNamespace, podName string) (string,
 	return capr.CompressInterface(logs)
 }
 
+// getPodFileContents executes a corresponding `kubectl cp` and gathers data for the purposes of helping increasing debug data.
+func getPodFileContents(podNamespace, podName, podPath string) (string, error) {
+	destFile := fmt.Sprintf("/tmp/%s", base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf("%s/%s/%s", podNamespace, podName, podPath))))
+	kcp := []string{
+		"-n",
+		podNamespace,
+		"cp",
+		fmt.Sprintf("%s:%s", podName, podPath),
+		destFile,
+	}
+
+	cmd := exec.Command("kubectl", kcp...)
+	if err := cmd.Run(); err != nil {
+		logrus.Errorf("error running kubectl -n %s cp %s:%s %s", podNamespace, podName, podPath, destFile)
+		return "", nil
```

Comment:
Should an error be returned here?
```suggestion
		logrus.Errorf("error running kubectl -n %s cp %s:%s %s", podNamespace, podName, podPath, destFile)
		return "", err
```

If so, do we need the log message, or can we leave it to the caller to log if necessary?